sql-queries-5/export-sql-schema-without-data/customize-export-mysql.sh

```diff
@@ -0,0 +1,2 @@
+$ mysqldump -u user -p -h 127.0.0.1 --no-data --skip-triggers University > university_schema_no_triggers.sql
```

Comment:
This script does not execute. Same for the other ones. Can you commit with chmod +x, and remove the $ signs from each script?